charts/linkerd2-cni/values.yaml

```diff
@@ -52,6 +48,14 @@ tolerations:
 # for more information
 #nodeAffinity:
 
+# -|- Image section
+image:
+  # -- Docker image for the CNI plugin
+  name: "cr.l5d.io/linkerd/cni-plugin"
+  # -- Tag for the CNI container Docker image
+  version: "v1.0.0"
+  imagePullPolicy:
```

Comment:
A good default here imo is `"IfNotPresent"`.  We set it on linkerd's values too. We should probably do the same here. I think it also helps with testing locally; by default the value will be `Always` so the kubelet will try to pull from ghcr instead of using cached images.

> By default, the kubelet tries to pull each image from the specified registry. However, if the imagePullPolicy property of the container is set to IfNotPresent or Never, then a local image is used (preferentially or exclusively, respectively).

https://kubernetes.io/docs/concepts/containers/images/#pre-pulled-images

Reply:
Thanks, fixed